### ISSUE TITLE
Wrong regular expression

### ISSUE THREAD

**Issue by LoneSurvivor82** [lite3/Unity-BitmapFontImporter#14]
Hi,

this is an excpert of the .fnt-file I tried to use your BitmapFontImporter on:

```
info face="D:\someFolders\Fonts\fontGUI_shoe-export" size=32 bold=0 italic=0 charset="" unicode=0 stretchH=100 smooth=1 aa=1 padding=0,0,0,0 spacing=1,1
common lineHeight=18 base=26 scaleW=98 scaleH=98 pages=1 packed=0 alphaChnl=1 redChnl=0 greenChnl=0 blueChnl=0
page id=0 file="D:\someFolders\Fonts\fontGUI_shoe-export.png"
chars count=67
char id=48 x=44 y=72 width=10 height=12 xoffset=0 yoffset=6 xadvance=11 page=0 chnl=0 letter=\"0\"
char id=49 x=91 y=28 width=6 height=12 xoffset=0 yoffset=6 xadvance=7 page=0 chnl=0 letter=\"1\"
char id=50 x=22 y=69 width=10 height=12 xoffset=0 yoffset=6 xadvance=11 page=0 chnl=0 letter=\"2\"
...
char id=89 x=0 y=15 width=10 height=14 xoffset=0 yoffset=4 xadvance=11 page=0 chnl=0 letter=\"Y\"
char id=90 x=0 y=0 width=10 height=14 xoffset=0 yoffset=4 xadvance=11 page=0 chnl=0 letter=\"Z\"
char id=32 x=0 y=0 width=0 height=0 xoffset=0 yoffset=0 xadvance=6 page=0 chnl=0 letter=\" \"
char id=9 x=0 y=0 width=0 height=0 xoffset=0 yoffset=0 xadvance=48 page=0 chnl=0 letter=\"	\"
```

kernings count=0 

It struggled with the last to lines with the space and the tab which gave some exceptions. I debugged it and found out, that your regular expression has a problem with that.
I modified your regular expression to the following and now it's working:

I changed FntParse.cs, line 124 from
`pattern = new Regex(@"\S+="".+?""|\S+");`
to
`pattern = new Regex(@"\S+=\\"".+\\""|\S+");`

**Comment by litefeel** [comment]
Hi.
Thanks for your feedback.
What software is used to export fnt file.

`pattern = new Regex(@"\S+=\\?"".+?\\?""|\S+");`

You can use this regular expression for compatibility.

**Comment by LoneSurvivor82** [comment]
Sorry for the long delay. I am using ShoeBox 3.6.5

**Comment by litefeel** [comment]
Fixed and Release to [3.2.0 ](https://github.com/litefeel/Unity-BitmapFontImporter/releases/tag/3.2.0)
https://github.com/litefeel/Unity-BitmapFontImporter/releases/tag/3.2.0
